tests/Tracing/Doctrine/DBAL/TracingDriverForV32Test.php

```diff
@@ -109,4 +109,45 @@ public function testGetExceptionConverter(): void
 
         $this->assertSame($exceptionConverter, $driver->getExceptionConverter());
     }
+
+    public function testCreateDatabasePlatform(): void
+    {
+        $databasePlatform = $this->createMock(AbstractPlatform::class);
+
+        $decoratedDriver = $this->createMock(DriverInterface::class);
+        $decoratedDriver->expects($this->once())
+            ->method('getDatabasePlatform')
+            ->willReturn($databasePlatform);
+
+        $driver = new TracingDriverForV32($this->connectionFactory, $decoratedDriver);
+
+        $this->assertSame($databasePlatform, $driver->createDatabasePlatformForVersion('5.7'));
+    }
+
+    public function testCreateDatabasePlatformForVersionWhenDriverDefinedCreateDatabasePlatformForVersion(): void
+    {
+        $databasePlatform = $this->createMock(AbstractPlatform::class);
+
+        $decoratedDriver = $this->createMock(StubCreateDatabasePlatformForVersionDriver::class);
+        $decoratedDriver->expects($this->once())
+            ->method('createDatabasePlatformForVersion')
+            ->with('5.7')
+            ->willReturn($databasePlatform);
+
+        $driver = new TracingDriverForV32($this->connectionFactory, $decoratedDriver);
+
+        $this->assertSame($databasePlatform, $driver->createDatabasePlatformForVersion('5.7'));
+    }
+}
+
+if (interface_exists(DriverInterface::class)) {
+    interface StubCreateDatabasePlatformForVersionDriver extends DriverInterface
+    {
+        /**
+         * @param string $version
+         *
+         * @return AbstractPlatform
+         */
+        public function createDatabasePlatformForVersion($version);
```

Comment:
Given that the method does not exist in the parent interface and when it will be added it will be typehinted anyway, it makes more sense to avoid the PHPDoc

```suggestion
        public function createDatabasePlatformForVersion(string $version): AbstractPlatform;
```

Reply:
✅